Python/optimizer.c

```diff
@@ -1194,6 +1202,7 @@ make_executor_from_uops(_PyUOpInstruction *buffer, int length, const _PyBloomFil
     executor->jit_code = NULL;
     executor->jit_side_entry = NULL;
     executor->jit_size = 0;
+    executor->run_count = UINT32_MAX;
```

Comment:
If this becomes a boolean, we can just make this `true`.

Also, can you add a comment explaining why we set this (so the executor isn't immediately detected as cold and wiped out)?